example/lib/meeting/hms_sdk_interactor.dart

```diff
@@ -20,12 +20,12 @@ class HMSSDKInteractor {
     hmsSDK.build();
   }
 
-  void join({required HMSConfig config}) async {
+  void join({required HMSConfig config}){
     this.config = config;
     hmsSDK.join(config: this.config);
```

Comment:
But join is an async function right?

Reply:
When we call join and the rest, that better to await them.

Reply:
@yahu1031  No, `join` function is not async. Once you call `join`, if it succeeds or fails, `onJoin` or `onHMSError` callbacks of `HMSUpdateListener` are invoked.

Reply:
But `hmsSDK.join()` is async right?

Reply:
@yahu1031 there is no need of await(async) in the join call because the success or error callbacks are not dependent on return values, they are sent using `onHMSError` & `HMSUpdateListener`.